02 Fx/01 React/README.md

```diff
@@ -2,242 +2,245 @@
+- Let's update `webpack.config`:
 
-### ./webpack.config.js
-```diff
-...
+  ### ./webpack.config.js
+  ```diff
+  ...
 
-module.exports = {
-  context: path.join(basePath, 'src'),
-  entry: {
-    app: './students.js',
--   appStyles: [
--     './mystyles.scss',
--   ],
+  module.exports = {
+    context: path.join(basePath, 'src'),
+    entry: {
+      app: './students.js',
+  -   appStyles: [
+  -     './mystyles.scss',
+  -   ],
+      ...
+    },
     ...
-  },
-  ...
-};
-
-```
-
-- React is an open source library for building user interfaces (quite popular nowdays), let's start by installing the library ([react](https://www.npmjs.com/package/react) and [react-dom](https://www.npmjs.com/package/react-dom)).
-
-```
-npm install react --save
-npm install react-dom --save
-```
-
-- Now, we can uninstall `jquery` because we don't need for this sample:
-
-```
-npm uninstall jquery
-```
-
-- And update `webpack.config` vendor stuff:
-
-### ./webpack.config.js
-```diff
-...
-
-module.exports = {
-  context: path.join(basePath, 'src'),
-  entry: {
-    app: './students.js',
-    vendor: [
--     'jquery',
-+     'react',
-+     'react-dom',
-    ],
-    vendorStyles: [
-      '../node_modules/bootstrap/dist/css/bootstrap.css',
-    ],
-  },
-  ...
-};
+  };
 
-```
+  ```
 
-- In the *`index.html`* file let's add a `<div>` that will be the entry
-point for our React app.
+- React is an open source library for building user interfaces, quite popular nowdays. Let's start by installing the library which is splitted into 2: [react](https://www.npmjs.com/package/react) as the core library and [react-dom](https://www.npmjs.com/package/react-dom) as the glue between React and the DOM.
 
-### ./src/index.html
-```diff
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Webpack 2.x by sample</title>
-  </head>
-  <body>
--   <div class="jumbotron">
--     <h1>Testing Bootstrap</h1>
--     <p>
--       Bootstrap is the most popular ...
--     </p>
--   </div>
--   Hello Webpack 2!
--  <div class="red-background ">
--    RedBackground stuff
--  </div>
-+  <div id="root">
-+  </div>
-  </body>
-</html>
-
-```
-
-- Let's create `AverageComponent` to create our first component under `src` folder:
-
-### ./src/averageComponent.jsx
-```javascript
-import * as React from 'react';
-import {getAvg} from './averageService';
-
-export class AverageComponent extends React.Component {
-  constructor() {
-    super();
-
-    this.state = {
-      scores: [90, 75, 60, 99, 94, 30],
-      average: 0,
-    };
-  }
+  ```
+  npm install react --save
+  npm install react-dom --save
+  ```
 
-  componentDidMount() {
-    this.setState({average: getAvg(this.state.scores)});
-  }
+- `jquery` won't be needed for this sample, so you can safely uninstall it:
+
+  ```
+  npm uninstall jquery
```

Comment:
Could you add `--save-dev` here? With that we can remove jquery from our `package.json`